src/main/java/thePackmaster/cards/conjurerpack/GeothermalHeating.java

```diff
@@ -0,0 +1,44 @@
+package thePackmaster.cards.conjurerpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.powers.intothebreachpack.AcidPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.applyToEnemy;
+
+public class GeothermalHeating extends ConjurerCard
+{
+    public final static String ID = makeID(GeothermalHeating.class);
+    private static final int BLOCK = 5;
+    private static final int MAGIC = 3;
+
+
+    public GeothermalHeating() {
+        super(ID, 1, CardType.SKILL, CardRarity.COMMON, CardTarget.ENEMY);
+        baseBlock = BLOCK;
+        baseMagicNumber = magicNumber = MAGIC;
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        blck();
+        if (m != null)
+        {
+            int count = 1;
```

Comment:
Starting this at 1 means that you'll always get additional block from this, even if the enemy has no debuffs. I think you can start at 0.

You can also remove the `m.powers.size() > 0` check right below this -- it isn't needed, since if `m.powers` is empty, the for loop will happen 0 times.